spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/integration/IntegrationProperties.java

```diff
@@ -295,4 +302,86 @@ public void setMessageMappingEnabled(boolean messageMappingEnabled) {
 
 	}
 
+	public static class Poller {
+
+		/**
+		 * Maximum of messages to poll per polling cycle.
+		 */
+		private Long maxMessagesPerPoll;
```

Comment:
This field could have a default value that matches `PollerMetadata.MAX_MESSAGES_UNBOUNDED`. That would then make the default available in auto-complete when configuring the property.